lib/board.rb

```diff
@@ -0,0 +1,22 @@
+class Board
```

Comment:
The Board class (as all classes) should have two types of properties:
**attributes** - you already have it, the @tiles. 
**behavior/methods** - these methods are directly related in taking in inputs from other classes it is interacting with and doing appropriate changes in its data/attributes because of these interactions. For instance the Board class maybe should have a method that accepts an index (this index is a specific player's choice of square) and makes an appropriate change in the tiles.